models/relation.rb

```diff
@@ -20,6 +20,7 @@
 module Models
   class Relation < Base
     ALL_OR_NONE_RELATIONS = %i[shipment meetup].freeze
+    ALTERNATIVE_COMPATIBLE = %i[order same_route sequence shipment force_first never_first force_end]
```

Comment:
ALTERNATIVE_COMPATIBLE_RELATIONS  (all others have _RELATIONS :)

Reply:
Can you .freeze it?

Reply:
Also a test would be nice (we have one for LINKING_RELATIONS and FORCING_RELATIONS for example) so that if we modify it by mistake the test will alert us.